packages/node/test/unit/plugins/operator/heartbeatUtils.test.ts

```diff
@@ -1,5 +1,6 @@
-import { createHeartbeatMessage, HeartbeatMessageSchema } from '../../../../src/plugins/operator/heartbeatUtils'
+import { omit } from 'lodash'
```

Comment:
Maybe

```ts
import omit from 'lodash/omit'
```

Reply:
@mondoreale Is this needed anymore when we start to use `rollup`? We have multiple `import { ... } from 'lodash'`  style imports currently in the codebase. If we change this, should we change all other occurences, too?